tim/config.py

```diff
@@ -45,4 +45,4 @@ def configure_sentry() -> str:
 
 
 def opensearch_request_timeout() -> int:
-    return int(os.getenv("OPENSEARCH_REQUEST_TIMEOUT", "30"))
+    return int(os.getenv("OPENSEARCH_REQUEST_TIMEOUT", "120"))
```

Comment:
I'm curious why the need to change the code for an env configurable value? I'm not against this change if this feels like a more appropriate default (I'm not sure if it does as I don't have the context that led to this change of the default)

Reply:
We don't actually currently configure this in env via terraform. It was coded as env-configurable so we could in the future if we end up wanting different timeouts for, say, dev vs. prod. For now as we're learning about OpenSearch a long default seems to make sense.

Reply:
If that's 120 seconds that seems really high as a default for a timeout. I'm not sure in what situations we're seeing a need for this high, but I also don't have a specific reason to block it other than 2 minutes is forever lol.

Reply:
Agreed, although 30 seconds was not long enough in dev...that was when we were doing simultaneous bulk ingests, which we aren't doing now, but still. I'd like to start with the baseline setup being "we know this works with our lowest tier OpenSearch instance" and then make adjustments to improve speed/efficiency from there, rather than the other way around.